starter-core/src/main/java/io/micronaut/starter/feature/lang/groovy/application.rocker.raw

```diff
@@ -34,11 +34,19 @@ class Application {
     static class DefaultEnvironmentConfigurer implements ApplicationContextConfigurer {
```

Comment:
I think we should rename static  this to be more agnostic. Please, note you could have one feature which requires eager initialization of singletons but does not specify a default environment.

Reply:
or just remove eager initialization from the ApplicationContextConfigurer as specified below